lib/Factory/DisplayFactory.php

```diff
@@ -120,6 +120,7 @@ public function getById($displayId, $showTags = false)
     public function getByLicence($licence)
     {
         $displays = $this->query(null, ['disableUserCheck' => 1, 'license' => $licence]);
+        $this->getLog()->debug('DISPLAY LOG LICENCE ' . $licence );
```

Comment:
if we need to leave this in, can it be reworded and not capitals?

Reply:
we don't need it, there is a chance I forgot to remove it.